head.html

```diff
@@ -1,4 +1,5 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
+<meta charset="UTF-8">
```

Comment:
Why remove the charset?

Reply:
"remove"? I'm adding it or am I missing something